changelog/fix-sl-233

```diff
@@ -1,4 +1,4 @@
 Significance: patch
 Type: tweak
 
-Increase payment failure correction timer to 60 seconds. [SL-233]
+Cache Seating service status checks. [SL-239]
```

Comment:
We need a new changelog entry instead of overwriting the one for SL-233. You can generate as many as you need by `npm run changelog`. This is using a automattic's jetpack changelogger which we include as a dev dependency in composer.